src/main/java/seedu/moneygowhere/parser/ConsoleParserConfigurations.java

```diff
@@ -51,6 +51,17 @@ public class ConsoleParserConfigurations {
     public static final String COMMAND_ADD_EXPENSE_ARG_CURRENCY_DESC = "currency";
     public static final boolean COMMAND_ADD_EXPENSE_ARG_CURRENCY_HAS_VAL = true;
     public static final boolean COMMAND_ADD_EXPENSE_ARG_CURRENCY_IS_MAND = false;
+    public static final String COMMAND_ADD_EXPENSE_ARG_MODE_OF_PAYMENT = "p";
+    public static final String COMMAND_ADD_EXPENSE_ARG_MODE_OF_PAYMENT_LONG = "mode of payment";
+    public static final String COMMAND_ADD_EXPENSE_ARG_MODE_OF_PAYMENT_DESC = ""
+            + "PayLah/PayNow/Cash/Card";
+    public static final boolean COMMAND_ADD_EXPENSE_ARG_MODE_OF_PAYMENT_HAS_VAL = true;
+    public static final boolean COMMAND_ADD_EXPENSE_ARG_MODE_OF_PAYMENT_IS_MAND = false;
+    public static final String COMMAND_ADD_EXPENSE_ARG_MODE_OF_PAYMENT_PAYLAH = "PayLah";
+    public static final String COMMAND_ADD_EXPENSE_ARG_MODE_OF_PAYMENT_PAYNOW = "PayNow";
+    public static final String COMMAND_ADD_EXPENSE_ARG_MODE_OF_PAYMENT_CASH = "Cash";
+    public static final String COMMAND_ADD_EXPENSE_ARG_MODE_OF_PAYMENT_CARD = "Card";
```

Comment:
Is it possible to rename the constant variables to COMMAND_ADD_EXPENSE_ARG_MODE_OF_PAYMENT_VAL_<VALUE> so that it will be easier to search the file for argument values?

Reply:
mm makes sense alright!